### ISSUE TITLE
Is it still equiprobable?

### ISSUE THREAD

**Issue by Schizo** [sylefeb/VoxModSynth#2]
I read in the last line of the Readme that this implementation is equiprobable, was wondering if that is still the case.

**Comment by sylefeb** [comment]
Yes, I unfortunately did not find the time to work more it. Contribs welcome!
